pages/index.tsx

```diff
@@ -21,6 +22,19 @@ export default function Home({
   return (
     <Layout>
       <div className="container">
+        <NextSeo
+          title="open source at ACM at UCLA"
+          description="at the largest computer science community at UCLA, we care about open-source"
+          openGraph={{
+            images: [{
+              url: 'https://www.uclaacm.com/images/logo.png',
```

Comment:
Could we use an image under this subdomain (ex `opensource.uclaacm.com) instead of relying on the external ACM website?

Reply:
Hey, thanks! Would using the logo.png file in the public folder work, or should I put a link in there?

Reply:
Oh wait, my mistake, it asks for a URL.

Reply:
There is already an image of our logo in `opensource.uclaacm.com`, if you look in `/public/logo.png`! Try linking to that as the URL!